cmd/check_container.go

```diff
@@ -168,17 +170,40 @@ func getContainerPolicyExceptions(ctx context.Context, pc pyxisClient) (policy.P
 
 func checkContainerPositionalArgs(cmd *cobra.Command, args []string) error {
 	if len(args) != 1 {
-		return fmt.Errorf("a container image positional argument is required")
+		return fmt.Errorf("a container image positional argument is required or a flag is not specified as --flag=value")
 	}
 
+	cmd.Flags().VisitAll(func(f *pflag.Flag) {
+		if f.Changed && f.Value.String() == "--submit" {
+			// We have --submit in one of the flags. That's a problem.
+			// We will set the submit flag to true so that the next block functions properly
+			submit = true
+		}
```

Comment:
I don't think this matches a case where a user did:

```
--pyxis-api-token --submit=true`
```

And I think that `--submit` is valid otherwise.

Reply:
If they did --submit=true, that should be caught in the regular flow. But I'll add a test for that just in case.

Reply:
You were right. I've added a test to cover this case. And updated the code to reflect that.